src/sentry/integrations/pagerduty/actions/form.py

```diff
@@ -43,25 +43,23 @@ def __init__(self, *args, **kwargs):
         self.fields["service"].choices = services
         self.fields["service"].widget.choices = self.fields["service"].choices
 
-    def _validate_service(self, service_id: int, integration_id: int | None) -> None:
+    def _validate_service(self, service_id: int, integration_id: int) -> None:
         params = {
             "account": dict(self.fields["account"].choices).get(integration_id),
             "service": dict(self.fields["service"].choices).get(service_id),
         }
 
-        try:
-            service = PagerDutyService.objects.get(id=service_id)
-        except PagerDutyService.DoesNotExist:
-            raise forms.ValidationError(
-                _('The service "%(service)s" does not exist in the %(account)s Pagerduty account.'),
-                code="invalid",
-                params=params,
-            )
-
-        integration = integration_service.get_integration(
-            organization_integration_id=service.organization_integration_id
+        ois = integration_service.get_organization_integrations(
```

Comment:
nit: I find these 2-3 letter variable names harder to follow than a more verbose variable at least denoting the type